core/testconnection.inc.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+define('APPROOT', dirname(__FILE__).'/../');
+require_once(APPROOT.'core/parameters.class.inc.php');
+require_once(APPROOT.'core/utils.class.inc.php');
+require_once(APPROOT.'core/restclient.class.inc.php');
+
+if (DIRECTORY_SEPARATOR === '\\') {
```

Comment:
This test can be better

Reply:
test and log are useless. i removed them